template.html

```diff
@@ -92,6 +73,28 @@ <h2><a href="#infi-test">
       </section>
 
       <section class="blocks">
+        <nav class="blocks-bar">
+            <button
+              title="{{utilityBar.toggleView}}"
+              class="summary-toggle"
+              data-toggle="summary"
+              role="switch"
+              aria-label="{{utilityBar.toggleView}}"
+              aria-expanded="true"
+            >
+              <div class="grid-view">
+                <div></div>
+                <div></div>
+                <div></div>
+                <div></div>
+              </div>
+              <div class="list-view">
+                <div></div>
+                <div></div>
+              </div>
+              <span class="sr-only">{{utilityBar.toggleView}}</span>
+            </button>
+        </nav>
```

Comment:
1. As you said, this button could have an explanatory text accompanying the icon. The space is also there for it.
2. You might want to use a `+` and `-` or chevron down / up icon for it now.

Reply:
I figured we can improve button styling in another PR. Maybe when we do a styling pass to unify the design more.